prov/efa/src/rdm/efa_rdm_ep_utils.c

```diff
@@ -137,9 +131,42 @@ struct efa_rdm_peer *efa_rdm_ep_get_peer(struct efa_rdm_ep *ep, fi_addr_t addr)
 struct efa_rdm_peer *efa_rdm_ep_get_peer_implicit(struct efa_rdm_ep *ep, fi_addr_t addr)
 {
 	struct efa_conn *conn;
```

Comment:
I think you can still keep the `efa_rdm_get_peer_impl` common function to avoid code duplication, you just need a bool to indicate whether it's an implicit av or not

Reply:
`efa_rdm_ep_get_peer` is called in the CQ read fast path and I am trying to minimize branching

Reply:
fair enough